src/constraint_system/plookup_composer.rs

```diff
@@ -469,7 +469,7 @@ mod tests {
         let mut prover = PlookupProver::new(b"demo");
```

Comment:
Looks good to me, could be worthwhile marking this is a test circuit. So that we can add all the different types of circuits into integration tests, for benchmarking later. If not, it can be done later - up to you.

Reply:
After some discussion with @lopeetall, we have decided to merge and port this issue to integration tests.